frontend/src/scenes/billing/billingLogic.ts

```diff
@@ -234,9 +234,15 @@ export const billingLogic = kea<billingLogicType>([
                         message: `You have exceeded the usage limit for ${productOverLimit.name}. Please 
                             ${productOverLimit.subscribed ? 'increase your billing limit' : 'upgrade your plan'}
                             or data loss may occur.`,
+                        dismissKey: 'usage-limit-exceeded',
                     }
                 }
 
+                localStorage.setItem(
+                    'components.lemon-banner.lemonBannerLogic.usage-limit-exceeded.isDismissed',
```

Comment:
This is weirdly tedious. I tried linking it directly to the lemonBannerLogic so I could get simply set the cookie through that logic. That led to weird "max call stack size exceeded" errors and simply didn't work.

Reply:
How did it not work? It should Just Work. Can you just set a different dismisskey based on the billing cycle end date?

Reply:
I added a "resetDismissKey" to the lemonbannerlogic and tried calling it here but the localstorage key never changed. It also resulted in replay related "max call stack size exceeded" errors.

Reply:
To be more clear, trying to set the dismissKey back to false through the lemonBannerLogic wasn't working. This localstorage.setItem implementation _does_ work.

Reply:
Ohhh I see what you are saying. I'd just append a date to the dismissKey, see comment on line 237